firefox-ios/Client/Frontend/Autofill/Address/AddressListView.swift

```diff
@@ -0,0 +1,61 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import SwiftUI
+import Common
+import Shared
+import Storage
+
+// MARK: - AddressListView
+
+/// A view displaying a list of addresses.
+struct AddressListView: View {
+    // MARK: - Properties
+
+    @Environment(\.themeType)
+    var themeVal
+    @ObservedObject var viewModel = AddressListViewModel()
```

Comment:
curious to see why we are initializing here. It seems we're injecting the viewmodel during creation, so do we need to initialize again? If not, then it seems we can also remove the profile being nil in the initializer. Would we want to show this view if profile is nil?

![image](https://github.com/mozilla-mobile/firefox-ios/assets/6743397/9561d391-188d-4412-bc78-a6432419f75e)

Reply:
Removed  = AddressListViewModel()